src/Package/Impl/Resources.resx

```diff
@@ -465,6 +465,9 @@
   <data name="HistoryWindowCaption" xml:space="preserve">
     <value>R History</value>
   </data>
+  <data name="ConnectionManagerWindowCaption" xml:space="preserve">
```

Comment:
Done

---
In reply to: [77055282](https://github.com/Microsoft/RTVS/pull/2237#discussion_r77055282) [](ancestors = 77055282)